packages/darklang/prettyPrinter/programTypes.dark

```diff
@@ -919,7 +924,9 @@ module Darklang =
         let signature (f: LanguageTools.BuiltinFunction) : String =
           let parameters =
             f.parameters
-            |> Stdlib.List.map (fun p -> $"({p.name} : {p.``type``})")
+            |> Stdlib.List.map (fun p ->
+                let params = PrettyPrinter.RuntimeTypes.typeReference p.``type``
+                $"({p.name} : {params})")
             |> Stdlib.String.join ", "
 
-          parameters ++ " -> " ++ f.returnType
\ No newline at end of file
+          parameters ++ " -> " ++ (f.returnType |> PrettyPrinter.RuntimeTypes.typeReference)
```

Comment:
TODO: re-check this
It seems weird to use the runtimeTypes pretty-printer here

Reply:
Hmm, it does feel a bit odd.

The only alternative I can think of would be updating RuntimeTypes (.fs) such that `BuiltInFn`'s  `returnType` (as well as `parameters`' type) had _PT_ `TypeReference`s.

The main blocker to _that_ is that "RT comes before PT" in `LibExecution` and we've had that line drawn in the sand for ages. I question that line pretty often, but it'd take some wrangling to get it all compiling again if we did that.

I'd put a CLEANUP here noting the weirdness and potential solution^ but otherwise fine for now.

---

Oh, another alternative would be introducing a RT2PT module _just_ for TypeReference, but that seems like a can of worms / slippery slope / some other idiom.